static/app/components/quickTrace/index.tsx

```diff
@@ -203,7 +212,7 @@ export default function QuickTrace({
     nodes.push(currentNode);
   }
 
-  if (children.length) {
+  if (children?.length) {
```

Comment:
Why'd this change?

Reply:
Orphan errors don't have children so children here can be undefined. We don't  return early in this component. We push to nodes and return at the end.